docs/source/tutorial/tutorial-authentication.mdx

```diff
@@ -14,7 +12,7 @@ In this section, you'll add the ability to log in to the example server and obta
 
 A **mutation** is an operation that changes state on your server. In this case, the mutation changes back-end state by creating a session tied to a particular user of your client.
 
-Open [GraphiQL](https://apollo-fullstack-tutorial.herokuapp.com/) and delete anything that's already in the left-hand panel. Open the Schema tab and select the `login` mutation:
+Open [GraphiQL](https://apollo-fullstack-tutorial.herokuapp.com/) and delete anything that's already in the left-hand panel. Open the Docs tab and select the `login` mutation:
```

Comment:
Good catch! I was using an older version of GraphiQL when I wrote this and that's where they were. I thought I got all these but apparently not - you'll see that in the screenshot it's still showing the Schema tab. Do you mind also updating the screenshot?

Reply:
Done & done! Appreciate the feedback!